README.md

```diff
@@ -380,6 +380,8 @@ For the debug feature, also run:
 npm run start:debugServer
 ```
 
+⚠️ Building monaco-vscode-api is only supported on Linux or Mac. It you use Windows, have a look at [WSL](https://learn.microsoft.com/fr-fr/windows/wsl/install) ⚠️
```

Comment:
You linked the french page. It does it automatically when going to the page. I get the german page.
Please use this:
https://learn.microsoft.com/en-us/windows/wsl/install

Reply:
Oh well spotted

Reply:
I though: "Oh la la, je ne comprends pas" 😆

Reply:
e09e563aad1e68d1723aebc209e6afc5d272d94b